build_modules/lib/src/kernel_builder.dart

```diff
@@ -63,14 +63,20 @@ class KernelBuilder implements Builder {
   /// If not provided, defaults to "lib/libraries.json" in the sdk directory.
   final String librariesPath;
 
+  /// The `--target` argument passed to the kernel worker.
+  ///
+  /// Optional. When omitted the [platform] name is used.
+  final String kernelTargetName;
+
   KernelBuilder(
       {@required this.platform,
       @required this.summaryOnly,
       @required this.sdkKernelPath,
       @required this.outputExtension,
       String librariesPath,
       bool useIncrementalCompiler,
-      String platformSdk})
+      String platformSdk,
+      this.kernelTargetName})
```

Comment:
nit: set this in the initializer so we don't have to deal with it being null anywhere else

Reply:
Done